docs/testplan/admin-ui.md

```diff
@@ -0,0 +1,53 @@
+| UX Search     | Search User by Email                                 | Customer opens a support ticket and notes down his email address. How easy is it to find the user in our database? |  |  
+|               | Search User by Project ID                            | Customer opens a support ticket and notes down his Project ID and forgets to mention his email address. Or we identify that a project is sharing illegal files. Project ID is still unique and should allow us to find the corresponding user. Next step could be an account freeze (to make sure the test returns the user and not the project). |  | 
+|               | Search User by User Agent                            | Customer has 2 user accounts. We know one of them has a user agent set and the other account is more for testing with no special settings. How easy can we identify the main account? |  |
+|               | Search User by Placement Constraint                  | Customer has 2 user accounts. We know one of them has a placement constraint set and the other account is more for testing with no special settings. How easy can we identify the main account? |  |
+|               | Search Project by Project ID                         | Customer would like to have a limit increase and gives us his project ID. |  | 
+|               | Search Project by Project Name                       | Customer gives us his email address and project name but forgets to mention his project ID. |  |
+|               | Search Project by User Agent                         | Customer has 2 projects. We know one of them has a user agent and the other one doesn't. How easy can we identify the important project? |  |
+|               | Search Project by Placement Constraint               | Customer has 2 projects. We know one of them has a placement constraint and the other one doesn't. How easy can we identify the important project? |  |
+|               | Search Bucket by Bucket Name                         | Customer has a special bucket name that is unique enough to search for it. We don't know the project ID or email address. Can we search for the bucket name without having to ask the customer for more details? |  |
+|               | Search Bucket by User Agent                          | We have special pricing bound to specific user agents. Can we search for all buckets that have this user agent set? |  |
+|               | Search Bucket by Placement Constraint                | We define a new placement constraint. After deployment we find out that we have a bug. Do we have any buckets with that placement constraint yet or can we just fix the bug without having to worry about possible side effects? |  |
```

Comment:
The current design has no section to search buckets without accessing the host project.